firefox/data/class_model.js

```
@@ -0,0 +1,278 @@
+var Gmail2Trello = Gmail2Trello || {};
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)